libsolidity/analysis/ControlFlowGraph.h

```diff
@@ -99,8 +99,7 @@ struct CFGNode
 	/// Exit nodes. All CFG nodes to which control flow may continue after this node.
 	std::vector<CFGNode*> exits;
 	/// Function call done by this node
-	FunctionCall const* functionCall = nullptr;
-
+	FunctionDefinition const* functionDefinition;
```

Comment:
```suggestion
	FunctionDefinition const* functionDefinition = nullptr;
```

Also, you should adjust the docstring. It should say that this is actually the resolved definition corresponding to that function call.

Reply:
done